apps/back/src/Expense.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class Expense {
+    private string $name;
+    private \DateTime $date;
+    private string $description;
+    private int $amount;
+    private array $participants;
+    private User $payer;
+
+    public function __construct(string $name, DateTime $date, string $description, int $amount, array $participants, User $payer)
+    {
+        $this->name = $name;
+        $this->date = $date;
+        $this->description = $description;
+        $this->amount = $amount;
+        $this->participants = $participants;
+        $this->payer = $payer;
+    }
```

Comment:
```suggestion
    public function __construct(
        private string $name,
        private DateTime $date,
        private string $description,
        private int $amount,
        private array $participants,
        private User $payer
    ) {
    }
```

Petite nouveauté avec PHP 8 : si ton constructeur ne fait qu'initialiser des propriétés à ta classe, tu peux les déclarer de cette façon. La notation est plus courte et tout aussi lisible. Voici [un petit article de blog](https://stitcher.io/blog/constructor-promotion-in-php-8) qui le présente de manière simple. 👍